.github/workflows/apply_api_variant.yml

```diff
@@ -0,0 +1,32 @@
+name: Apply API variant
+
+on: push
+
+jobs:
+  standard_api_project:
+    name: Standard API project
+    uses: nimblehq/elixir-templates/.github/workflows/reusable_phoenix_project.yml@composite_2.0
```

Comment:
I think we can simplify it like this, according to Github [document](https://docs.github.com/en/actions/using-workflows/reusing-workflows#calling-a-reusable-workflow)
> `./{path}/{filename}` for reusable workflows in the same repository.

```suggestion
    uses: ./.github/workflows/reusable_phoenix_project.yml@composite_2.0
```

Reply:
I'm not sure about this but `./.github/workflows/reusable_phoenix_project.yml@composite_2.0` it doesn't work @longnd, the workflow doesn't start 😢

![image](https://user-images.githubusercontent.com/11751745/153568932-65eb7777-8d06-416e-aa1e-efe9b8508b40.png)

Reply:
![image](https://user-images.githubusercontent.com/11751745/153569261-73844761-8454-428a-9abb-0595ead2c961.png)

Reply:
That why I have to revert it back to `nimblehq/elixir-templates/`

![image](https://user-images.githubusercontent.com/11751745/153569356-aa43af07-eb6b-4ad2-becb-aaea11946390.png)

Reply:
@andyduong1920 I think by referring to the file using a relative path, we can't specify the version, it means we can only do like this

```suggestion
    uses: ./.github/workflows/reusable_phoenix_project.yml@composite_2.0
```

but this isn't a big deal. I'm fine with the current approach 🤞

Reply:
@andyduong1920 I think by referring to the file using a relative path, we can't specify the version, it means we can only do like this

```suggestion
    uses: ./.github/workflows/reusable_phoenix_project.yml@composite_2.0
```

but this isn't a big deal. I'm fine with the current approach 🤞